xbmc/cores/AudioEngine/DSPAddons/ActiveAEDSP.cpp

```diff
@@ -513,7 +513,7 @@ CAEChannelInfo CActiveAEDSP::GetInternalChannelLayout(AEStdChLayout stdLayout)
   return CAEUtil::GetAEChannelLayout(channelLayoutOut);
 }
 
-bool CActiveAEDSP::CreateDSPs(unsigned int &streamId, CActiveAEDSPProcessPtr &process, AEAudioFormat inputFormat, AEAudioFormat outputFormat, bool upmix,
+bool CActiveAEDSP::CreateDSPs(unsigned int &streamId, CActiveAEDSPProcessPtr &process, AEAudioFormat &inputFormat, AEAudioFormat &outputFormat, bool upmix,
```

Comment:
Could something break when e.g. CActiveAEDSP::CreateDSPs(...) modify inputFormat? 
In all methods the format is copied to internal variables.